addons/cni-canal/canal.yaml

```diff
@@ -4551,9 +4573,16 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: FLANNELD_IFACE
+              {{ if eq .Config.ClusterNetwork.IPFamily "IPv4+IPv6" }}
```

Comment:
Why is this needed?

Reply:
Because of the {{ else }} part. Wanted to preserve existing behavior in non dualstack case. 

```

              valueFrom:
                fieldRef:
                  fieldPath: status.hostIP
```

Reply:
ah OK, got it